### PR TITLE
Changes to publish to pip

### PR DESCRIPTION
**Pull request by howellhou**
1. Added a setup file 
2. Restructured some files.
3. Updated the README file.

Tested on the test pypi environment: https://test.pypi.org/project/cyanobyte/0.0.15/

author and author_name left blank for now

### CODE REVIEW

**Comment by howellhou** [comment]
@googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ It can also generate reference documentation for a periphreal, useful for embedd
 This is not an official Google product.
 
 ## Setup
-Note: This project requires Python3.
+Note: This project requires Python3. You can install using pip
 
-`pip install -r requirements.txt --user`
+`pip install cyanobyte`
```
Comment (Fleker) [inline README.md:11]: We probably still want a section a bit lower on development setup rules.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="cyanobyte",
+    version="0.0.1",
+    #author="",
```
Comment (Fleker) [inline setup.py:9]: Google Inc.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    #author_email="",
```
Comment (Fleker) [inline setup.py:10]: fleker@google.com for now I suppose

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    description="A package that generates library files for a peripheral given an intermediary layer (YAML files)",
+    long_description=long_description,
+    long_description_content_type=" text/markdown",
```
Comment (Fleker) [inline setup.py:13]: Does this have an extra space

File: README.md [hunk 5]
```diff
@@ -6,15 +6,15 @@ It can also generate reference documentation for a periphreal, useful for embedd
...
 
 For more advanced development, also install the dev list.
```
Comment (Fleker) [inline README.md:13]: And this step would be associated with development

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    url="https://github.com/google/cyanobyte",
+    packages=setuptools.find_packages(exclude=("test",)),
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: Apache Software License",
+        "Operating System :: OS Independent",
+    ],
+    python_requires=">=3.0",
+    install_requires=[
```
Comment (Fleker) [inline setup.py:22]: Does this list intend to have versions as well? Is it possible to pull from requirements.txt directly for maintainability?

File: setup.py (hunk 2 above)
Comment (Fleker) [inline setup.py:9]: Google Inc

File: setup.py (hunk 3 above)
Comment (Fleker) [inline setup.py:10]: fleker@google.com for now

File: setup.py (hunk 4 above)
Comment (Fleker) [inline setup.py:13]: Should there be a space before text/markdown?

**Pull request merged**